go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/dell/csm-replication
 
-go 1.23
+go 1.23.0
```

Comment:
is this needed, as we maintain major versions

Reply:
Addressed

Reply:
@nitesh3108  The checks are failing with go 1.23.

Reply:
I am noticing the same in csi-powerstore PR.
My concern is that updating the patch version to '0' would make the code use the specific go release (1.23.0).